plugins/Goals/Controller.php

```diff
@@ -128,14 +128,11 @@ protected function getGoalReportView($idGoal = false)
         $view->nameGraphEvolution = 'Goals.getEvolutionGraph' . $idGoal;
         $view->topDimensions = $this->getTopDimensions($idGoal);
 
-        // conversion rate for new and returning visitors
-        $segment = urldecode(\Piwik\Plugins\VisitFrequency\API::RETURNING_VISITOR_SEGMENT);
```

Comment:
This caused the performance problems in the past. Also further down the `visitorType==new`. We do now make sure those are pre-archived.